blog.qmd

```diff
@@ -0,0 +1,329 @@
+         pattern = str_to_title(pattern) |> str_replace_all('_', '-')) 
+  
+```
+
+```{r}
+#| echo: false
+courses_with_keywords %>% 
+  select(school, discipline, course, course_credits,  sentence, pattern) |>
+  arrange(school, pattern, discipline) %>% 
+  gt() %>% 
+  urbn_gt_theme() %>% 
+  opt_interactive(use_text_wrapping = FALSE, use_resizers = TRUE, use_compact_mode = TRUE) 
+```
+
+Having identified the keywords in the course descriptions, we now turn our attention to quantifying the prevalence of different types of WBL opportunities across schools. To do this, we group our data by school and pattern (which represents the type of WBL opportunity), count the number of occurrences, and then reshape our data so that each type of WBL opportunity becomes a separate column. We also add a 'total' row that sums up the counts across all schools for each type of opportunity.
+
+```{r}
+freq_table <-
+  courses_with_keywords %>%
+  group_by(school, pattern) %>%
+  count() %>%
+  group_by(pattern) %>%
+  bind_rows(summarise(.,
+                      across(where(is.numeric), sum),
+                      across(where(is.character), ~ "total"))) %>% 
+  ungroup() %>%
+  pivot_wider(names_from = pattern, values_from = n) %>%
+  mutate(across(where(is.numeric), ~ replace_na(.x, 0))) %>% 
+  slice(n(), 1:n()-1)
+```
+
+```{r}
+#| echo: false
+freq_table %>%
+  rename_with(~str_to_title(.x), everything()) %>% 
+  gt() %>% 
+  urbn_gt_theme() %>% 
+  tab_header(title = "Frequency of Work-Based Learning Opportunities by School")
+```
+
+## Discussion
+
+Our text analysis has provided valuable insights into the prevalence of WBL opportunities in Florida's community colleges. The medical and education sectors dominate WBL opportunities, with clinicals and field experiences being the most common types. However, apprenticeships, despite their proven value in career preparation and advancement, are one of the least offered forms of WBL.
+
+Our analysis revealed that the medical and education sectors dominate WBL opportunities, which is expected given that these fields often require practical experience for licensure. Clinicals, largely offered in medical and related fields, and field experiences, predominantly in the education sector, were the most common types of WBL identified. However, we also found that apprenticeships, despite their value in career preparation and advancement ([Kuehn, Marotta, et al. 2022](https://www.urban.org/research/publication/beyond-productivity-how-employers-gain-more-apprenticeship)), are one of the least offered forms of WBL.
+
+It's important to note that our methodology has limitations. For instance, our current dataset lacks information on the quality of WBL experiences and dimensions of equity, such as whether the programs are paid. This is a significant consideration since many community college students work while enrolled in their courses and may not be able to devote time to an unpaid internship or other WBL program.
```

Comment:
I think one more limitation would help round this out, e.g. in the dictionaries we compiled, the sample of schools we could consider, the course descriptions themselves, etc.